### PR TITLE
Add new flag "initial-state" for init command

### PR DESCRIPTION
**Pull request by renuka-fernando**
Fixes: https://github.com/wso2/product-apim-tooling/issues/161
#### Happy path
Command:
```sh
apictl init pets --oas=products_swagger.yaml --initial-state=PUBLISHED
```
Output:
```log
Initializing a new WSO2 API Manager project in /Users/renuka/Desktop/pets
Project initialized
Open README file to learn more
```

#### Invalid initial state
Command:
```sh
apictl init pets --oas=products_swagger.yaml --initial-state=DEPRECATED
```
Output:
```log
apictl: Invalid initial API state: DEPRECATED
Valid initial states: [PUBLISHED]
Exit status 1
```

### CODE REVIEW

File: import-export-cli/cmd/init.go [hunk 1]
```diff
@@ -327,5 +351,7 @@ func init() {
 		"YAML definition of API")
 	InitCommand.Flags().StringVarP(&initCmdSwaggerPath, "oas", "", "", "Provide an OpenAPI "+
 		"specification file for the API")
+	InitCommand.Flags().StringVar(&initCmdInitialState, "initial-state", "", "Provide the initial state "+
```
Comment (dinusha92) [inline import-export-cli/cmd/init.go]: @renuka-fernando Have we considered the possibility of using "lc-state"? IMO, "lc-state" is short. But "lc" it self wouldn't make any sense to a new user. If that's the case we can come up with something else?  I feel like we can come up with something better than "initial-state". Just a thought.

File: import-export-cli/utils/constants.go [hunk 2]
```diff
@@ -105,3 +105,5 @@ const ApplicationId = "applicationId"
 const ApiId = "apiId"
 const DefaultCliApp = "default-apictl-app"
 const DefaultTokenType = "JWT"
+
+var ValidInitialStates = []string{"PUBLISHED"}
```
Comment (dinusha92) [inline import-export-cli/utils/constants.go]: If someone put initial-state as "CREATED". It would get invalidated if we put only "PUBLISHED" here.

Comment (dinusha92) [reply import-export-cli/utils/constants.go]: @pubudu538

Comment (malinthaprasan) [reply import-export-cli/utils/constants.go]: @renuka-fernando can we fix this? we should allow CREATED and PUBLISHED both

**Pull request merged**